clients/gtest/gemm_gtest.yaml

```diff
@@ -150,10 +150,10 @@ Definitions:
     - { M:   256, N: 24001, K:   256, lda:   256, ldb: 24000, ldc: 24040, ldd: 24040 }
 
   - &alpha_beta_range
-    - { alpha:  5, beta:  0 }
-    - { alpha:  0, beta:  3 }
-    - { alpha:  1, beta:  3 }
-    - { alpha:  1, beta:  1 }
+    - { alpha:  5, beta:  0, alphai:  0, betai: -3 }
+    - { alpha:  0, beta:  3, alphai: -3, betai:  0 }
+    - { alpha:  1, beta:  3, alphai:  0, betai:  4 }
+    - { alpha:  1, beta:  1, alphai:  4, betai:  0 }
```

Comment:
It is okay if `alpha`, `alpha`, `beta`, `betai` are specified in the same combination. Basically all of the mappings here get merged into the mapping for the test case, and whichever gets merged last, wins. There is no reason to separate `alpha` and `alphai`, but they can be separated if necessary.